File: exercises/Python/chapter_3/9_series_of_integers.py
"""
9. Series of Integers
Design a Python program to read a series of integers. The first integer is special, as it indicates how many more integers
will follow. The output of the program will be the calculated the sum and average of the integers [excluding the first
integer].
"""

n = int(input("How many numbers to be read? "))
total = 0

for i in range(n):
    next_number = int(input("Enter a number: "))
    total += next_number

if n > 0:
    print("Sum:", total)
    print("Average:", total / n)
else:
    print("You've decided not to input any number.")

File: exercises/C/chapter_3 - Loops/9_series_of_integers.c
// 9. Series of Integers
// Design a Python program to read a series of integers. The first integer is special, as it indicates how many more integers
// will follow. The output of the program will be the calculated the sum and average of the integers [excluding the first
// integer].
#include <stdio.h>


int main(int argc, char const *argv[])
{
    int n, number, sum = 0;
    double avg;

    printf("How many numbers to be entered? ");
    scanf("%d", &n);

    for (int i = 0; i < n; i++)
    {
        printf("Enter a number: ");
        scanf("%d", &number);

        sum += number;
    }

    if (n > 0)
    {
        avg = sum / n;
        printf("%d numbers read.\n", n);
        printf("Sum: %d\n", sum);
        printf("Average: %.1lf\n", avg);
    }
    else
        printf("Input Stream Closed!!!\n");
    return 0;
}
